bfsysroot/bfunwind/CMakeLists.txt

```diff
@@ -20,8 +20,12 @@ cmake_minimum_required(VERSION 3.6)
 project(bfunwind C CXX)
 include(${BF_FLAGS_DIR}/flags.cmake)
 
-set(CMAKE_ASM_NASM_OBJECT_FORMAT "elf64")
```

Comment:
This is very cut / paste. Can we create a function or macro for this?

Reply:
Yeah, no problem - will be a little while before I get it done but I'll amend the PR.